server/api/endpoints/shop_endpoints/orders.py

```diff
@@ -345,11 +345,17 @@ def patch(
         id=order.id,
     )
 
-    if str(updated_order.shop_id) == "d3c745bc-285f-4810-9612-6fbb8a84b125":
-        account = account_crud.get(updated_order.account_id)
-        post_discord_order_complete(
-            f"New order from {account.name}", settings=co2_shop_settings, order=updated_order, email=account.name
-        )
+    try:
+        shop = load(ShopTable, updated_order.shop_id)
+        if shop.discord_webhook is not None:
+            account = account_crud.get(updated_order.account_id)
+            settings = DiscordSettings(BOT_NAME=shop.name, WEBHOOK_URL=shop.discord_webhook)
```

Comment:
normally the settings classes are used to:
- provide an easy way to set default
- provide an easy way to override "settings" via the env

This "misuses" the settings class to "organise" some vars. A normal dict or pydantic class can also work.

https://docs.pydantic.dev/latest/concepts/pydantic_settings/

Probably easier to just change the function signature of `post_discord_order_complete` so it accepts BOTNAME/WEBHOOK URL and remove the settings class completely.